### DISCUSSION TITLE
Allow to vendor:publish stubs

### DISCUSSION THREAD

**Discussion by RiaanZA** [studio1902/statamic-peak-commands#4] [Ideas]
The default "add block" template stub contains css classes that don't fit in with my theme. It will speed up my dev process if I can edit the stub to use classes matching my layout instead of having the need to edit them every time I create a new block

Is it possible to allow this (and other stubs) to be published to the stubs folder?

**Comment by robdekort** [comment]
I get the idea and it's something I was happy to give up and get update support for in return. I'd gladly merge a PR but don't want to do the time investment myself.

**Comment by marcorieser** [comment]
@RiaanZA Will do my best to add this in #7

**Reply by robdekort** [reply]
Amazing Marco! Seems to work brilliantly.

**Comment by marcorieser** [comment]
@RiaanZA You can now publish the stubs with `php artisan vendor:publish --tag=statamic-peak-commands-stubs`.

**Reply by RiaanZA** [reply]
Amazing, Thanks @marcorieser for the quick response!
